.github/workflows/ecosystem-submission.yml

```diff
@@ -125,63 +100,167 @@ jobs:
+    - name: PR create comment
       uses: peter-evans/create-or-update-comment@v3
       with:
-        issue-number: ${{ github.event.issue.number }}
-        body: |
-          Tests with latest version of Qiskit release passed! :sparkles:
+        issue-number: ${{ steps.cpr.outputs.pull-request-number }}
+        body-path: "summary_output.md"
 
-    - name: Create comment on failure for stable check
-      if: ${{ steps.check-return.outputs.PASS_STB != 'True' }}
-      uses: peter-evans/create-or-update-comment@v3
+    # Issue labels and comment
+    - name: Issue labeling status ready
+      uses: actions/github-script@v6
+      if: ${{ steps.check-return.outputs.PASS_STD == 'True' }}
       with:
-        issue-number: ${{ github.event.issue.number }}
-        body: |
-          Tests with latest version of Qiskit release failed! :warning:
-          This means your project doesn't work with the latest version of Qiskit.
-          This is purely informational and doesn't affect your project joining the Ecosystem, but you may want to investigate the problem.
-          See logs: https://github.com/qiskit-community/ecosystem/actions/runs/${{ github.run_id }}
+        script: |
+          github.rest.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ["ready"]
+          })
+    - name: PR labeling status ready remove on hold
+      continue-on-error: true
+      uses: actions/github-script@v6
+      if: ${{ steps.check-return.outputs.PASS_STD == 'True' }}
+      with:
+        script: |
+          github.rest.issues.removeLabel({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            name: ["on hold"]
+          })
 
-    - name: Create comment on success for dev check
-      if: ${{ steps.check-return.outputs.PASS_DEV == 'True' }}
-      uses: peter-evans/create-or-update-comment@v3
+    - name: Issue labeling status on hold
+      uses: actions/github-script@v6
+      if: ${{ steps.check-return.outputs.PASS_STD != 'True' }}
       with:
-        issue-number: ${{ github.event.issue.number }}
-        body: |
-          Tests with development version of Qiskit release passed! :sparkles:
+        script: |
+          github.rest.issues.addLabels({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            labels: ["on hold"]
+          })
+    - name: PR labeling status on hold remove ready
+      continue-on-error: true
+      uses: actions/github-script@v6
+      if: ${{ steps.check-return.outputs.PASS_STD != 'True' }}
+      with:
+        script: |
+          github.rest.issues.removeLabel({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            name: ["ready"]
+          })
```

Comment:
Is this duplicate of 194-205?

Reply:
Not here is for the issue, on line 170 is for the PR